cli/cmd/create.go

```diff
@@ -56,20 +57,20 @@ func runCreate(
 	args []string,
 ) error {
 	stackName := args[0]
-	happyClient, err := makeHappyClient(cmd, sliceName, stackName, []string{tag}, createTag, dryRun, ModeCreate)
+	happyClient, err := makeHappyClient(cmd, sliceName, stackName, []string{tag}, createTag, ModeCreate)
 	if err != nil {
 		return errors.Wrap(err, "unable to initialize the happy client")
 	}
 
-	ctx := cmd.Context()
+	ctx := context.WithValue(cmd.Context(), options.DryRunKey, dryRun)
```

Comment:
dry run becomes a context value. most of this program doesn't need to know about it. For the pieces that do, pull it from the context

Reply:
Last round of dryrun changes was causing stacks to be deleted after --dry-run was completing; do you see side effects?

Reply:
This PR breaks `happy create <STACK_NAME> --dry-run`.